sosa.ml

```diff
@@ -1335,6 +1346,9 @@ module List_of (Char: BASIC_CHARACTER) :
   let rev t = List.rev t
 
   let fold t ~init ~f = List.fold_left t ~init ~f
+  let foldi t ~init ~f = 
+    snd (List.fold_left t ~init:(0,init)
```

Comment:
Not a big deal, and I don't know what's common, but might there be a space after a comma within tuples?